### PR TITLE
8-1-23 updates

### PR DESCRIPTION
**Pull request by nsesodia**
Clean up & optimization of functions. New feature on Home Page

### CODE REVIEW

File: .idea/workspace.xml [hunk 1]
```diff
@@ -132,5 +156,10 @@
   </component>
   <component name="VcsManagerConfiguration">
     <option name="ADD_EXTERNAL_FILES_SILENTLY" value="true" />
+    <MESSAGE value="Organized image files. Updated pathes" />
+    <MESSAGE value="Optimized Car &amp; Mission Statement sections to be reusable between pages." />
+    <MESSAGE value="Cleaned Up Header.css file &amp; global variables" />
+    <MESSAGE value="Component created for optimal reuse" />
+    <option name="LAST_COMMIT_MESSAGE" value="Component created for optimal reuse" />
   </component>
 </project>
```
Comment (Seth10001) [inline .idea/workspace.xml:165]: What is this `.idea/workspace.xml` file?

Comment (Seth10001) [reply .idea/workspace.xml:165]: Oh I see it's from jetbrains

File: src/pages/About.js [hunk 2]
```diff
@@ -36,137 +35,85 @@ let p3 = "Inclusivity is one of our founding principles. We believe that everyon
...
+//                 </Col>
+//             )
+//         }
+//     }
+//     else {
+//         return (
+//             <DividedSection img={img} blurb={blurb} blurb2={blurb2} img_align={img_align}/>
+//         )
+//     }
+// }
```
Comment (Seth10001) [inline src/pages/About.js:116]: There's a couple commented out components here, do you plan on bringing/deleting them back before merging?

File: src/components/CarsSection/CarsSection.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import Container from 'react-bootstrap/Container'
+import Row from 'react-bootstrap/Row'
+import Button from "react-bootstrap/Button";
+import {Link} from "react-router-dom";
+import car1 from "../../content/assets/images/car_pics/torch/torch_ontrack_clean.jpg";
+import car2 from "../../content/assets/images/car_pics/cielo/IMG_9219.JPG";
```
Comment (Seth10001) [inline src/components/CarsSection/CarsSection.js:7]:
```suggestion
import car2 from "../../content/assets/images/car_pics/Cielo/IMG_9219.JPG";
```

File: src/components/CarsSection/CarsSection.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import Container from 'react-bootstrap/Container'
+import Row from 'react-bootstrap/Row'
+import Button from "react-bootstrap/Button";
+import {Link} from "react-router-dom";
+import car1 from "../../content/assets/images/car_pics/torch/torch_ontrack_clean.jpg";
```
Comment (Seth10001) [inline src/components/CarsSection/CarsSection.js:6]:
```suggestion
import car1 from "../../content/assets/images/car_pics/Torch/torch_ontrack_clean.jpg";
```

File: src/pages/Cars.js [hunk 5]
```diff
@@ -1,18 +1,23 @@
...
-import "../content/styles/Header.scss";
-import "../content/styles/Cars.scss";
-import "../content/styles/Button.scss";
-import torch from "../content/assets/images/car_pics/Torch/torch_team.JPG";
-import sunrider from "../content/assets/images/car_pics/car3.jpg";
-import cielo from "../content/assets/images/car_pics/car2.JPG";
 import Button from "react-bootstrap/Button";
 import {Link} from "react-router-dom";
+import Header from "../components/Header";
+import torch from "../content/assets/images/car_pics/torch/torch_ontrack_clean.jpg";
```
Comment (Seth10001) [inline src/pages/Cars.js:8]:
```suggestion
import torch from "../content/assets/images/car_pics/Torch/torch_ontrack_clean.jpg";
```

File: src/pages/Cars.js [hunk 6]
```diff
@@ -1,18 +1,23 @@
...
+import sunrider from "../content/assets/images/car_pics/sunrider/2021/car3.jpg";
+import cielo from "../content/assets/images/car_pics/cielo/IMG_9011_crop.JPG";
```
Comment (Seth10001) [inline src/pages/Cars.js:10]:
```suggestion
import cielo from "../content/assets/images/car_pics/Cielo/IMG_9011_crop.JPG";
```

**Comment by Seth10001** [comment]
@nsesodia I loaded this locally and this is looking FRESH 🔥

File: src/pages/About.js (hunk 2 above)
Comment (nsesodia) [reply src/pages/About.js:116]: I commented it out when I was creating a reusable component, must have forgotten to go back and delete the commented section. Feel free to remove it.

**Review by matlshen — APPROVED** [review]

**Pull request merged**